Description: Override upstream copy and use libbsd
  Avoid using local copy of strlcpy, use the libbsd library in overlay mode
  instead on our supported targets.
Author: Yuri D'Elia <wavexx@thregr.org>
Forwarded: not-needed
Updated: 2019-10-24

--- a/Makefile.linux
+++ b/Makefile.linux
@@ -1,5 +1,6 @@
-CPPFLAGS += -D_GNU_SOURCE -D_LINUX_PORT -Imissing
+CFLAGS += -D_GNU_SOURCE $(shell pkg-config --cflags libbsd-overlay) -Imissing
+LDFLAGS += -lpthread $(shell pkg-config --libs libbsd-ctor)
 MANPREFIX ?= ${PREFIX}/share/man
-EXTRA_SRC = missing/strlcpy.c missing/kqueue_inotify.c
+EXTRA_SRC = missing/kqueue_inotify.c
 
 include Makefile.bsd
